awx/main/tasks/receptor.py

```diff
@@ -639,11 +639,11 @@ def kube_config(self):
 #
 RECEPTOR_CONFIG_STARTER = (
     {'local-only': None},
-    {'log-level': 'info'},
+    {'log-level': settings.RECEPTOR_LOG_LEVEL},
     {'node': {'firewallrules': [{'action': 'reject', 'tonode': settings.CLUSTER_HOST_ID, 'toservice': 'control'}]}},
     {'control-service': {'service': 'control', 'filename': '/var/run/receptor/receptor.sock', 'permissions': '0660'}},
     {'work-command': {'worktype': 'local', 'command': 'ansible-runner', 'params': 'worker', 'allowruntimeparams': True}},
-    {'work-signing': {'privatekey': '/etc/receptor/signing/work-private-key.pem', 'tokenexpiration': '1m'}},
+    {'work-signing': {'privatekey': '/etc/receptor/work_private_key.pem', 'tokenexpiration': '1m'}},
```

Comment:
Is this change to be _paired_ with https://github.com/ansible/awx-operator/pull/1442?

The new location looks like it's the correct one, I agree with the change, but I'd like a little bit better linking and description of the changes here, since it's not all all that this should happen from the title of this PR, and the associated changes & issue are not immediately discoverable.

Reply:
Never mind my prior comments, this was rendered defunct by https://github.com/ansible/awx/pull/14151, but did not show due to a git issue. This line is not actually changed with this merge.